db_conn.py

```diff
@@ -3,12 +3,12 @@
 
 # ADD URI STRING HERE #################################
 # set production False, if you want to contribute to this repo and make changes
-production = True
+production = False
```

Comment:
Is this required? It would better be `True`.

Reply:
Yes, this should be `True`.

Reply:
@vikasnitd Set the production variable to True .. and send the commit again